gems/smithy-client/lib/smithy-client/protocols/rpc_v2.rb

```diff
@@ -36,26 +36,24 @@ def error(context)
           errors_module.error_class(code).new(context, message, data)
         end
 
-        # def stub_data(operation, data)
-        #   resp = HTTP::Response.new
-        #   resp.status_code = 200
-        #   resp.headers['Smithy-Protocol'] = 'rpc-v2-cbor'
-        #   resp.headers['Content-Type'] = 'application/cbor'
-        #   resp.body = @codec.serialize(operation.output, data)
-        #   resp
-        # end
-
-        # def stub_error(operation, error_code)
-        #   resp = HTTP::Response.new
-        #   resp.status_code = 400
-        #   resp.headers['Smithy-Protocol'] = 'rpc-v2-cbor'
-        #   resp.headers['Content-Type'] = 'application/cbor'
-        #   type = operation.errors.find { |e| e.type.name.include?("Types::#{error_code}") }
-        #   shape = operation.errors.find { |e| e.id == type.id }
-        #   data = { '__type' => type.id, 'message' => 'stubbed-error-message' }
-        #   resp.body = @codec.serialize(shape, data)
-        #   resp
-        # end
+        def stub_data(operation, data)
+          resp = HTTP::Response.new
+          resp.status_code = 200
+          resp.headers['Smithy-Protocol'] = 'rpc-v2-cbor'
+          resp.headers['Content-Type'] = 'application/cbor'
+          resp.body = @codec.serialize(operation.output, data)
+          resp
+        end
+
+        def stub_error(error_code)
+          resp = HTTP::Response.new
+          resp.status_code = 400
+          resp.headers['Smithy-Protocol'] = 'rpc-v2-cbor'
+          resp.headers['Content-Type'] = 'application/cbor'
+          data = { '__type' => error_code, 'message' => 'stubbed-error-message' }
```

Comment:
Technically error code should be full shape id + error code, but in stub, it can just be error code, and the split('#').last logic still works. We could make this as real as possible but it requires parsing out the namespace. We may not even know the namespace of the error code anyway because it could be different than the service's.